src/app/routes/radioPodcast/getInitialData/index.js

```diff
@@ -15,19 +15,29 @@ import withRadioSchedule from '#app/routes/utils/withRadioSchedule';
 import getRadioService from '../../utils/getRadioService';
 import processRecentEpisodes from '../../utils/processRecentEpisodes';
 
-const DEFAULT_TOGGLE_VALUE = { enabled: false, value: 4 };
-
 const getRadioScheduleData = path(['radioScheduleData']);
 const getScheduleToggle = path(['onDemandRadioSchedule', 'enabled']);
-const getRecentEpisodesToggle = pathOr(DEFAULT_TOGGLE_VALUE, [
-  'recentAudioEpisodes',
-]);
 
 export default async ({ path: pathname, pageType, service, toggles }) => {
   try {
-    const onDemandRadioDataPath = overrideRendererOnTest(pathname);
+    const detailPageType = pathname.includes('podcast')
+      ? 'Podcast'
+      : 'On Demand Radio';
+    const isPodcast = detailPageType === 'Podcast';
+    const RADIO_PODCAST_MISSING_FIELD = isPodcast
+      ? PODCAST_MISSING_FIELD
+      : RADIO_MISSING_FIELD;
+    const DEFAULT_TOGGLE_VALUE = { enabled: false, value: isPodcast ? 8 : 4 };
+    const recentEpisodesKey = isPodcast
+      ? 'recentPodcastEpisodes'
+      : 'recentAudioEpisodes';
+    const getRecentEpisodesToggle = pathOr(DEFAULT_TOGGLE_VALUE, [
+      recentEpisodesKey,
+    ]);
```

Comment:
Might be more readable to pass these values from the route file, eg:

```javascript
export default config => async ({ path: pathname, pageType, service, toggles }) => {
    const { detailPageType, missingFieldCode, etc } = config;
```

Or to keep the variants together here, eg:

```javascript
const config = pathname.includes('podcast') ? {
    detailPageType: 'Podcast',
    missingFieldCode: PODCAST_MISSING_FIELD,
    defaultToggleValue: { enabled: false, value: 8 },
    recentEpisodesKey: 'recentPodcastEpisodes`
} : {
    // Radio Values
}
```